### DISCUSSION TITLE
https://demonkingswarn.is-a.dev/blog/2023/10/12/epic-games-unreal-engine-charge

### DISCUSSION THREAD

**Discussion by giscus** [DemonKingSwarn/blog#2] [General]
# https://demonkingswarn.is-a.dev/blog/2023/10/12/epic-games-unreal-engine-charge

https://demonkingswarn.is-a.dev/blog/2023/10/12/epic-games-unreal-engine-charge

**Comment by whinee** [comment]
testing giscus

# I AM SHOUTING

does markdown work tho

```python
print("hello world")
```

**Reply by DemonKingSwarn** [reply]
can confirm, it works

**Comment by RaghavGohil** [comment]
Yo cool blog thing! Keep it up!

**Reply by DemonKingSwarn** [reply]
thanks raghav-kun, very cool

**Comment by trueauracoral** [comment]
Very good article, your website is coming along

**Comment by shivamNgr** [comment]
Nice
